BrushedMotorControllerFirmware/include/driver.c

```diff
@@ -14,6 +14,8 @@
 
 #define PINS_PER_CHANNEL 2
 
+#define FULL_SPIN_DEGREES (360u * CONFIG_POSITION_CONTROL_MODIFIER)
```

Comment:
maybe DEGREES_PER_ROTATION ?

Reply:
Agreed, sounds better, corrected in https://github.com/eSqadron/NRF52840-Brushed-Motor-Controller-Firmware/pull/13